src/modules/pos/validator/components/ValidatorProfile/ValidatorProfile.css

```diff
@@ -373,22 +373,6 @@
         width: 100%;
         margin-bottom: -45px;
       }
-
-      &.delegateActive {
-        padding-right: 10px;
-      }
-
-      &.delegateStandby {
-        padding-right: 10px;
-      }
-
-      &.delegatePunished {
-        padding-right: 10px;
-      }
-
-      &.delegateBanned {
-        padding-right: 10px;
-      }
```

Comment:
@oskarleonard These styles were removed but not replaced? Any particular reason for that?

Reply:
This [scoped commit](https://github.com/LiskHQ/lisk-desktop/pull/4762/commits/f5e26669c47a6fe6fdcdcce3f720e541e8b6ae41) should explain it. In the design there shouldnt be any padding of these icons. And instead of renaming them i removed that logic